fingerprint_report_templates/fingerprint_report/templates/macros/macros.html

```diff
@@ -0,0 +1,87 @@
+{# TODO: OLD set of macros to be removed #}
```

Comment:
I would not name the file `macros.html` and put it in a directory called `macros`. It seems redundant and the naming doesn't explain what the file is about (especially when you have other files in the same `macros` directory like: `builders` and `renderers`)